### DISCUSSION TITLE
Has Runelite patched this out? I'm unable to locate the config.json

### DISCUSSION THREAD

**Discussion by Steelio** [Arnuh/RuneLiteHijack#8] [General]
Title says it all essentially.
my %userprofile%/.runelite folder doesn't contain a config.json anywhere.

Am I being dumb and looking in the wrong place, or has Runelite patched this?
Thanks for any response! <3

**Comment by Arnuh** [comment]
The config.json file still seems to exist for me but I did notice an issue with the latest launcher(2.6.0 or above it seems) and have a fix for it

![image](https://user-images.githubusercontent.com/2877803/231906405-43066690-0405-41ad-b3f7-9cd193413641.png)

are the new tested against versions

**Reply by Steelio** [reply]
Edit: Found it. Im just dumb and was looking in the wrong place. Thank you for the reply still! <3
